### PR TITLE
Cloud Log Manager and Syslog forwarders

### PR DESCRIPTION
**Pull request by tommaso-ascani**
UI parts:
- New modals on active loki cards
- New card on cluster status
- Some other fix

Actions:
- Added cloud log manager and syslog info to `list-loki-istances` api

[NethServer/dev#6932](https://github.com/NethServer/dev/issues/6932)

### CODE REVIEW

File: core/ui/src/components/settings/CloudLogManagerConfigureModal.vue [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+  mixins: [carbonPrefixMixin, TaskService, DateTimeService],
+  data() {
+    return {
+      radioVal: "last_timestamp",
+      toggleEnabled: false,
+      address: "",
+      tenant: "",
+      date: "",
+      time: "",
+      last_timestamp_formatted: "",
```
Comment (andre8244) [inline core/ui/src/components/settings/CloudLogManagerConfigureModal.vue:164]: Let's use camelCase for variables: `lastTimestampFormatted`

File: core/ui/src/components/settings/CloudLogManagerConfigureModal.vue [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+      }
+
+      if (this.configuration.address == "") {
+        this.address = "https://nar.nethesis.it";
+      } else {
+        this.address = this.configuration.address;
+      }
+
+      if (this.configuration.tenant == "") {
+        this.tenant = "clienteit";
```
Comment (andre8244) [inline core/ui/src/components/settings/CloudLogManagerConfigureModal.vue:212]: Just asking: is this the correct default value?

File: core/ui/src/components/settings/CloudLogManagerConfigureModal.vue [hunk 3]
```diff
@@ -0,0 +1,326 @@
...
+        this.createModuleTaskForApp(this.lokiToEdit.instance_id, {
+          action: taskAction,
+          data: {
+            active: this.active,
+            address: this.address,
+            tenant: this.tenant,
+            start_time: this.start_time,
+          },
+          extra: {
+            isNotificationHidden: true,
```
Comment (andre8244) [inline core/ui/src/components/settings/CloudLogManagerConfigureModal.vue:285]: `isNotificationHidden: true` should be used only for read actions. Use this as reference: https://github.com/NethServer/ns8-core/blob/main/core/ui/src/components/nodes/SetFqdnModal.vue#L218-L220

File: core/ui/src/components/settings/SyslogConfigureModal.vue [hunk 4]
```diff
@@ -0,0 +1,369 @@
...
+          data: {
+            active: this.active,
+            address: this.address,
+            port: this.port,
+            protocol: this.protocolVal,
+            format: this.formatVal,
+            start_time: this.start_time,
+          },
+          extra: {
+            isNotificationHidden: true,
```
Comment (andre8244) [inline core/ui/src/components/settings/SyslogConfigureModal.vue:328]: `isNotificationHidden: true` should be used only for read actions. Use this as reference: https://github.com/NethServer/ns8-core/blob/main/core/ui/src/components/nodes/SetFqdnModal.vue#L218-L220

File: core/ui/src/components/settings/LokiCard.vue [hunk 5]
```diff
@@ -177,4 +211,9 @@ export default {
 .margin-top {
   margin-top: 0.5rem;
 }
+
+.ns-error {
```
Comment (andre8244) [inline core/ui/src/components/settings/LokiCard.vue:215]: This CSS class can be removed, it's already inside `_core.scss`

File: core/ui/src/views/ClusterStatus.vue [hunk 6]
```diff
@@ -829,4 +954,9 @@ export default {
 .card-row:last-child {
   margin-bottom: 0;
 }
+
+.ns-error {
```
Comment (andre8244) [inline core/ui/src/views/ClusterStatus.vue:958]: This CSS class can be removed, it's already inside `_core.scss`

**Review by andre8244 — CHANGES_REQUESTED** [review]

File: core/ui/src/components/settings/CloudLogManagerConfigureModal.vue (hunk 2 above)
Comment (tommaso-ascani) [reply core/ui/src/components/settings/CloudLogManagerConfigureModal.vue:212]: No, this is only for Nethesis

**Comment by tommaso-ascani** [comment]
Per i vari 'Lorem Ipsum' parlo con Davide?

File: core/ui/src/components/settings/CloudLogManagerConfigureModal.vue (hunk 2 above)
Comment (andre8244) [reply core/ui/src/components/settings/CloudLogManagerConfigureModal.vue]: So I guess an empty string is better here

File: core/ui/src/components/settings/CloudLogManagerConfigureModal.vue [hunk 7]
```diff
@@ -0,0 +1,318 @@
...
+          action: taskAction,
+          data: {
+            active: this.active,
+            address: this.address,
+            tenant: this.tenant,
+            start_time: this.start_time,
+          },
+          extra: {
+            title: this.$t("action." + taskAction),
+            isNotificationHidden: false,
```
Comment (andre8244) [inline core/ui/src/components/settings/CloudLogManagerConfigureModal.vue:277]:
```suggestion
            description: this.$t("common.processing"),
```

**Review by andre8244 — APPROVED** [review]
Let's set notification description too, then we should be good

File: core/ui/src/components/settings/SyslogConfigureModal.vue [hunk 8]
```diff
@@ -0,0 +1,357 @@
...
+            active: this.active,
+            address: this.address,
+            port: this.port,
+            protocol: this.protocolVal,
+            format: this.formatVal,
+            start_time: this.start_time,
+          },
+          extra: {
+            title: this.$t("action." + taskAction),
+            isNotificationHidden: false,
```
Comment (andre8244) [inline core/ui/src/components/settings/SyslogConfigureModal.vue:316]:
```suggestion
            description: this.$t("common.processing"),
```

**Review by andre8244 — APPROVED** [review]

**Pull request merged**